pkg/controller/migplan/validation.go

```diff
@@ -247,10 +249,46 @@ func (r ReconcileMigPlan) validateNamespaces(plan *migapi.MigPlan) error {
 		})
 		return nil
 	}
+	namespaces := r.validateNamespaceLengthForDVM(plan)
+	if len(namespaces) > 0 {
+		plan.Status.SetCondition(migapi.Condition{
+			Type:     NsLengthExceeded,
+			Status:   True,
+			Reason:   LengthExceeded,
+			Category: Warn,
```

Comment:
should we make this critical?

if length >59 and configmap does not have a domain, migplan has a critical condition and is not ready?

Reply:
I preferred not to prevent the migplan from running since this case should be rare... but I suppose it's fair to say we should be raising a critical condition since the route will likely fail to get created

Reply:
We agreed to keep this as a warning for now on scrum. I confirmed it is displayed in the UI at the final screen of the wizard.